README.md

```diff
@@ -1,10 +1,86 @@
 # CARTA Frontend
+The CARTA frontend is used in conjunction with the [CARTA backend](https://github.com/CARTAvis/carta-backend), and for general use a pre-built production frontend is usually installed alongside the backend automatically. 
+However, a user may wish to install the frontend component separately in order to test beta versions, for example, or to develop it.
+
+## Pre-built production packages
+The simplest way to install the frontend component is to use the pre-built production packages from NPM (for any OS with Node.js installed), our PPA (Debian packages for Ubuntu), and our RPM repository (for RHEL based systems).
+
+### NPM
+
+Pre-built production frontends can be obtained from the [npmjs.com carta-frontend repository](https://www.npmjs.com/package/carta-frontend?activeTab=explore).
+
+If you have Node.js and `npm` already installed on your system, you can install a production frontend using the following command:
+
+```
+npm i carta-frontend
+```
+The files will be installed to your `node-modules` directory. For a system-wide 'global' installation (using the `-g` flag) it would be at `/usr/lib/node_modules`, whereas for a local user installation, it would be at `$PWD/node_modules`.
+
+You can install specific frontend versions by appending `@` plus the version number. For example,
+```
+npm i carta-frontend@1.4.0
+npm i carta-frontend@2.0.0-beta.0
+```
+The available versions are listed in the [npmjs.com carta-frontend versions tab](https://www.npmjs.com/package/carta-frontend?activeTab=versions).
+
+Alternatively, the pre-built production frontend can be downloaded directly without npm. For example,
+```
+wget https://registry.npmjs.org/carta-frontend/-/carta-frontend-2.0.0.tgz
+```
+The standalone `tgz` archive file extracts as `package/build` in your current directory.
+
+You can start the CARTA backend with a non-default frontend by specifying its location with the `--frontend_folder` flag. For example,
+```
+./carta-backend --frontend_folder $HOME/package/build
+```
+
+### PPA
+
+A Personal Package Archive (PPA) is a third-party package repository for Ubuntu. We supply pre-built production frontends at the [CARTAvis-team PPA on the Ubuntu Lauchpad](https://launchpad.net/~cartavis-team/+archive/ubuntu/carta).
+
+To install the latest release version:
+```
+sudo add-apt-repository ppa:cartavis-team/carta
+sudo apt-get update
+sudo apt-get install carta-frontend
+```
+
+The Ubuntu package install location for the frontend is `/usr/share/carta/frontend/`.
+
+(Once the PPA is added, you can also install the `carta` metapackage that includes both the CARTA backend and frontend components, as well as well as Beta versions of all the packages e.g. `carta-beta`, `carta-frontend-beta` and `carta-backend-beta`).
```

Comment:
I would replace this sentence with:

"The PPA also provides a `carta` metapackage which automatically installs the CARTA frontend and backend packages. In addition to the latest release versions, beta versions are provided in a parallel set of packages."